init_node.sh

```diff
@@ -94,6 +94,7 @@ RPC_PORT=26657
 P2P_PORT=26656
 PROF_PORT=6060
 GRPC_PORT=9090
+GRPC_WEB_PORT=9091
```

Comment:
Is `9091` official port number?

Reply:
Yes, default grpc-web port number in cosmos-sdk is 9091.